compiler/damlc/lib/DA/Cli/Options.hs

```diff
@@ -205,13 +205,13 @@ projectOpts name = ProjectOpts <$> projectRootOpt <*> projectCheckOpt name
                help "Check if running in Daml project."
             <> long "project-check"
 
-enableScenarioOpt :: Parser EnableScenarioService
-enableScenarioOpt = EnableScenarioService <$>
-    flagYesNoAuto "scenarios" True "Enable/disable support for running scenarios" idm
+enableScenarioServiceOpt :: Parser EnableScenarioService
+enableScenarioServiceOpt = EnableScenarioService <$>
+    flagYesNoAuto "scenarios" True "Enable/disable support for running Daml Scripts and scenarios" idm
 
-enableScriptsOpt :: Parser EnableScripts
-enableScriptsOpt = EnableScripts <$>
-    flagYesNoAuto "daml-script" True "Enable/disable support for running Daml Scripts" internal
+enableScenariosOpt :: Parser EnableScenarios
+enableScenariosOpt = EnableScenarios <$>
+    flagYesNoAuto "enable-scenarios" True "Enable/disable support for running scenarios." internal
```

Comment:
Can we have a comment that explains the difference between `--scenarios` and `--enable-scenarios`? The flags and descriptions are very similar, but they do different things, so it would be nice to have a comment for us.

Reply:
yeah, I wasn't sure how to phrase this better, do you or @cocreature have something in mind?

Reply:
`--scenarios` controls whether we start the scenario/script service and only that (no compile errors). We can probably point that out in the comment explicitly

The new flags control whether scenarios are supported as a language feature (it’s literally a compile error). Maybe drop the `running` from the help text to make it clear that this is the case.

Reply:
thanks! I've rephrased in the last commit, please let me know if the new descriptions are good.

Reply:
Nice, thank you!